m4/curl-compilers.m4

```diff
@@ -941,6 +956,14 @@ AC_DEFUN([CURL_SET_COMPILER_WARNING_OPTS], [
             tmp_CFLAGS="$tmp_CFLAGS -Wstrict-aliasing=3"
           fi
           #
+          dnl Only gcc 4.1 or later (possibly earlier)
+          if test "$compiler_num" -ge "401"; then
+            CURL_ADD_COMPILER_WARNINGS([tmp_CFLAGS], [shadow])
+            CURL_ADD_COMPILER_WARNINGS([tmp_CFLAGS], [unused])
+            CURL_ADD_COMPILER_WARNINGS([tmp_CFLAGS], [no-multichar])
+            CURL_ADD_COMPILER_WARNINGS([tmp_CFLAGS], [no-system-headers])
```

Comment:
This one is actually already defined below.

Reply:
You're right, thanks for spotting them.